silverorange/package_release
# Mono Repo Support

To test:

**Test with a non-monorepo.**
1. Check this PR out.
1. Go into `/so/sites/emrap/live`
1. Run `/so/packages/package_release/work-yourname/bin/prepare-site`
1. Ensure that everything looks correct.
1. Follow the on-screen instructions to check out live and delete the branch.

**Test with a monorepo**
1. Check this PR out.
1. Go into `/so/sites/emrap-corependium/live/client`
1. Run `/so/packages/package_release/work-yourname/bin/prepare-site`
1. Ensure that everything looks correct.
1. Follow the on-screen instructions to check out live and delete the branch.
1. Go into `/so/sites/emrap-corependium/live/server`
1. Run `/so/packages/package_release/work-yourname/bin/prepare-site`
1. Ensure that everything looks correct.
1. Follow the on-screen instructions to check out live and delete the branch.
